### PR TITLE
Addition of new estimation for incubation period from ambiguous dates of exposure

### PR DESCRIPTION
**Pull request by thibautjombart**
This will be useful for the North Kivu outbreak. Creating the PR to finalise this contribution.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: epitrix
 Title: Small Helpers and Tricks for Epidemics Analysis
-Version: 0.2.2
+Version: 0.3.0
 Authors@R: c(person("Thibaut", "Jombart", email = "thibautjombart@gmail.com", role = c("aut", "cre")),
              person("Anne", "Cori", email = "a.cori@imperial.ac.uk", role = c("aut")),
              person("Zhian N.", "Kamvar", email = "zkamvar@gmail.com", role = "ctb"),
```
Comment (thibautjombart) [inline DESCRIPTION:6]: @ffinger can you add yourself here?

**Comment by thibautjombart** [comment]
Also can you add a small description of the new functions in the 'main features' of the package listed in the `README.Rmd` (and compile to update `README.md`)?

File: DESCRIPTION (hunk 1 above)
Comment (ffinger) [reply DESCRIPTION:6]: am there already, 2 lines below

**Comment by ffinger** [comment]
> Also can you add a small description of the new functions in the 'main features' of the package listed in the `README.Rmd` (and compile to update `README.md`)?

Will do, will send another PR (since I can't modify this one)

**Comment by thibautjombart** [comment]
> > Also can you add a small description of the new functions in the 'main features' of the package listed in the `README.Rmd` (and compile to update `README.md`)?
> 
> Will do, will send another PR (since I can't modify this one)

Argh damn, sorry about that. Will close this one then. THanks :)

**Pull request closed without merging**